packages/lending/src/protocols/morphoblue/configs.ts

```diff
@@ -3,7 +3,7 @@ import * as logics from '@protocolink/logics';
 import { mainnetTokens } from './tokens';
 
 export const ID = 'morphoblue';
-export const DISPLAY_NAME = 'Morpho Blue';
+export const DISPLAY_NAME = 'Morpho';
```

Comment:
I thought the DISPLAY_NAME is the same as the folder name. If we would like to use Morpho, should we change the folder name too?

Reply:
`DISPLAY_NAME` is only used on the front end for display purposes. A similar case is `zeroex-v4`, where if we needed to set a display name for it, it would be `0x`, which doesn't necessarily match the folder name.

And I believe 'Morpho Blue' is still the brand name, and they just use 'Morpho' for the lending, so I don't think we need to change the folder name unless they decide to rebrand.